### DISCUSSION TITLE
migrating a whole instance to a new domain.

### DISCUSSION THREAD

**Discussion by simonheb** [mastodon/mastodon#20135] [General]
Assuming we have an an instance of some thousand users and would decide to move this to a new name because the initially chosen name was not ideal. Is there a way to migrate all users as an admin without requiring each user to migrate individually?

**Comment by tuetenk0pp** [comment]
This is completeley out of the blue: Maybe you could set up a new server and work with the ALTERNATE_DOMAINS key? 

https://docs.joinmastodon.org/admin/config/#basic
